index.html

```diff
@@ -5,278 +5,566 @@
+                fill="#CFCFCF"
+              />
+            </svg>
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/imtupai566">
+            <svg
+              width="20"
+              height="20"
+              viewBox="0 0 20 20"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+                fill="#CFCFCF"
+              />
+            </svg>
+          </a>
+        </li>
+        <li>
+          <a href="https://medium.com/@tupai566">
+            <svg
+              width="20"
+              height="16"
+              viewBox="0 0 20 16"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
+                fill="#CFCFCF"
+              />
+            </svg>
+          </a>
+        </li>
+      </ul>
+      <div class="headline">
+        <h1>Hey There.</h1>
+        <h1>I’m Tupai.</h1>
+        <h2>I’m a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website. Look through some
```

Comment:
Nice job so far👍. Let's try to make some adjustments to the positioning and spacing around each element so that your project can match the design😉. don't forget we are required in this milestone to ensure we stick to the given design👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="436" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/d3577d3c-d4f7-41a8-ade7-dc5ab2ab0dfc">|<img width="431" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/fa2c4a48-24fe-4948-b8d9-734b0ff16c20">|


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/d445c0e7-532f-4614-8342-51c9a73f13fe">|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/b3cb26e1-bfef-4e9e-bab6-c61fedbc6525">|


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/3519dccd-fcb3-4dd7-b044-c2a0c1bf9eee">|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/a8aa98e7-ba6b-42e8-8734-75cf1267a184">|


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/6314c5b2-46fe-48f8-afbf-871a389424ca">|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/31c57195-a314-4ea1-b2af-17da12dc6269">|


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/e23f35e4-ae45-4079-a9aa-553657f280e4">|<img width="450" alt="image" src="https://github.com/Tupai566/Day3/assets/64643995/bf4aef8b-ca8b-4fd6-b263-f09e8ab1efbe">|